scss/app.scss

```diff
@@ -327,6 +370,12 @@ form {
     border-left: 5px solid #b10e1e;
```

Comment:
I know this hasn't changed but is there a colour variable that could be used in place of this hex code?

Reply:
Updated this.